examples/run-queue.php

```diff
@@ -1,38 +1,35 @@
 <?php
+declare( strict_types = 1 );
 
 require_once dirname( __DIR__ ) . '/vendor/autoload.php';
 require_once __DIR__ . '/db-connect.php';
 
-global $capsule;
-
 echo '=== Push Notification Server ===' . PHP_EOL;
 
-$kid = getenv('KEY_ID');
-$tid = getenv('TEAM_ID');
-$key = getenv('KEY');
+$kid = strval( getenv( 'KEY_ID' ) );
```

Comment:
`getenv` can return `false`. One thing we could add is a global function to get this from the `.env` file with some validation to help folks running this to debug their `.env` file, but that can be down the road.

For now this is a kinda ugly quick-fix to get it to pass validation

Reply:
Fair enough. I think wrapping in some code that performs validation makes sense, as per #3. It's definitely a different piece of work, not something for this PR.